packages/webpack-cli/lib/utils/arg-parser.js

```diff
@@ -38,11 +38,11 @@ const argParser = (options, args, argsOnly = false, name = '') => {
     // Prevent default behavior
     parser.on('command:*', () => {});
 
-    // Use customized help output if available
-    parser.on('option:help', () => {
+    // Use customized help output
+    if (args.includes('--help')) {
```

Comment:
Do wee need https://github.com/webpack/webpack-cli/blob/master/packages/webpack-cli/lib/bootstrap.js#L25?

Reply:
Good catch. It should be removed from there.

Reply:
The same should be done for `version` too.

![Screenshot at 2020-10-20 16-20-20](https://user-images.githubusercontent.com/46647141/96576776-49878000-12f0-11eb-86aa-2caf79401030.png)

Reply:
I will update 👍🏻